tests/apollo/test_skvbc_view_change.py

```diff
@@ -318,17 +318,16 @@ class SkvbcViewChangeTest(unittest.TestCase):
 
         bft_network.start_all_replicas()
 
-        initial_primary = 0
-        
-        replcas_to_stop = [ v for v in range(initial_primary,
-                                             initial_primary + num_consecutive_failing_primaries) ]
-        
-        expected_final_primary = initial_primary + num_consecutive_failing_primaries
+        initial_primary = await bft_network.get_current_primary()
+        replcas_to_stop = [ v for v in range((initial_primary % bft_network.config.n),
+                                             (initial_primary % bft_network.config.n) + num_consecutive_failing_primaries) ]
         
+        expected_final_view = (initial_primary + num_consecutive_failing_primaries)
+
         await self._send_random_writes(tracker)
 
         await bft_network.wait_for_view(
-            replica_id=initial_primary,
+            replica_id=initial_primary % bft_network.config.n,
```

Comment:
again, I don't understand why you add `% bft_network.config.n`, given that `initial_primary = await bft_network.get_current_primary()`